### ISSUE TITLE
Build neural net blocks

### ISSUE THREAD

**Issue by kleincode** [kleincode/Llama2.jl#1]
- lines 182-215 in llama2.c: rmsnorm, softmax
- lines 338-345 in llama2.c: SwiGLU
